.claude/skills/hkj-spring/SKILL.md

```
@@ -220,6 +220,15 @@ public Validated<List<String>, User> createUser(@RequestBody UserRequest req) {
 }
 ```
 
+~~~admonish tip title="Sparse PATCH: `UpdateSpec`"
```

markdownlint-cli2 (0.23.0)

[warning] Code fence style
Expected: backtick; Actual: tilde

(MD048, code-fence-style)